content/en/blog/2025/otel-arrow-phase-2.md

```diff
@@ -0,0 +1,66 @@
+---
+title: {{ replaceRE "[-_]" " " .Name | title }}
+linkTitle: OpenTelemetry Protocol with Apache Arrow Phase 2
+date: {{ dateFormat "2006-01-02" .Date }}
```

Comment:
I think the shortcodes are what's breaking the build, so let's try just hardcoding the title and date. Also the link title (if used) should be a shortened version of the post title.

```suggestion
title: OpenTelemetry Protocol with Apache Arrow: Phase 2 Announcement
linkTitle: OTel-Arrow Phase 2
date: 2025-04-29
```